Marlin/src/module/planner.cpp

```diff
@@ -1934,21 +2141,33 @@ void Planner::_buffer_steps(const int32_t (&target)[XYZE]
       // NOTE: Computed without any expensive trig, sin() or acos(), by trig half angle identity of cos(theta).
       if (junction_cos_theta > 0.999999) {
         // For a 0 degree acute junction, just set minimum junction speed.
-        vmax_junction = MINIMUM_PLANNER_SPEED;
+        vmax_junction_sqr = sq(MINIMUM_PLANNER_SPEED);
```

Comment:
Fairly trivial, but having MINIMUM_PLANNER_SPEED_SQR in config (or calculated from it, not sure how that works) would save on this calculation.

Reply:
I think the compiler will optimize the sq operation ... Let me check ... 
In Arduino.h

#define sq(x) ((x)*(x))

So the compiler does the computation at compile time, no runtime overhead 👍

Reply:
Sorry  - I work mostly with managed code, as such have learnt never to trust the compiler optimizations :)

Reply:
Don´t worry. At least here, it will be optimized. But, there are other cases that will be not.